connectors/golden-tables/src/test/scala/io/delta/golden/GoldenTables.scala

```diff
@@ -378,6 +378,7 @@ class GoldenTables extends QueryTest with SharedSparkSession {
 
     val commitInfoFile = CommitInfo(
       version = Some(0L),
+      None,
```

Comment:
this is unrelated, but currently broken on the master. We don't compile when `connectors` when `spark` changes.